utils/listenForMessages.ts

```diff
@@ -45,8 +61,18 @@ export const listenForMessages = (user_id: string, dispatch: AppDispatch) => {
         dispatch(setTempHistory(message.res.V_TEMP));
       } else if (command === 'connected') {
         dispatch(addDevice(message.res.device));
+        const alertMessage: Alert = {
+          variant: 'informative',
+          text: 'Nowe urządzenie',
+        };
+        dispatch(addAlert(alertMessage));
       } else if (command === 'disconnected') {
         dispatch(removeDevice(message.res.device));
+        const alertMessage: Alert = {
+          variant: 'error',
+          text: 'Odłączono urządzenie',
```

Comment:
[general comment] It would be cool if we would show more details about the action, e.g. which device has been connected or disconnected, which value has been changed etc.

Reply:
Good idea but if I want to expand this functionality more I need to get device from slice. Did I use `selectDeviceWithId` in a acceptable way?
`const state = store.getState();`
`const device = selectDeviceWithId(state, location, nodeId, deviceId);`